pubspec.yaml

```
@@ -18,20 +18,26 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.15.1 <3.0.0"
 
+# Dependencies specify other packages that your package needs in order to work.
+# To automatically upgrade your package dependencies to the latest versions
+# consider running `flutter pub upgrade --major-versions`. Alternatively,
+# dependencies can be manually updated by changing the version numbers below to
+# the latest version available on pub.dev. To see which dependencies have newer
+# versions available, run `flutter pub outdated`.
 dependencies:
   carousel_pro: ^1.0.0
```

This package is not null safe, hence the app can't boot if we have this as our dependency. I found a quick fix on the internet - 
```suggestion
  carousel_pro_nullsafety: ^1.0.2
```
After this replace the `carousel_pro` import with `carousel_pro_nullsafety` and check if everything works.